.github/workflows/action-types.yml

```
@@ -0,0 +1,13 @@
+name: Validate action typings
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  validate-typings:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: krzema12/github-actions-typing@v0
```

After recent changes, it should say

```suggestion
      - uses: typesafegithub/github-actions-typing@v1
```